### ISSUE TITLE
Delete whole local file cache

### ISSUE THREAD

**Issue by DecaTec** [nextcloud/android#9128]
### Is your feature request related to a problem? Please describe.
I'm missing a feature for deleting the whole file cache of the Android app. When working with many files, these often needs to get downloaded in the app. These downloaded files remain in the local cache.
In order to clear the cache, you have to navigate to "on this device" in the app's menu and delete all folders shown (be careful that you only delete the cache and not the files in the cloud).

### Describe the solution you'd like
The app for iOS has a feature in its settings to delete the whole file cache of the app by only one tap.
I would love to see such a feature for the Android app.

### Describe alternatives you've considered
As described above, delete all cached folders from "on this device".

**Comment by ThomasHineXYZ** [comment]
Any update on this?...
The Nextcloud app is using ~10GB of "user data" and I need to clear space from my phone.

**Comment by AndyScherzinger** [comment]
@ThomasHineXYZ than just do it? This feature is there forever but as a Standard OS mechanism. You need to navigate to Apps and than Nextcloud via your Phones general settings. Than Android offers you the options to purge the cache or the user data.

**Comment by ThomasHineXYZ** [comment]
@AndyScherzinger Last time I tried that, it asked me to go through the whole rigmarole of signing in, waiting, and then re-syncing and re-setting up everything. I have some other apps set to load files from the Nextcloud app directly, such as Keepass. It also breaks all of the add-on companion apps such as Nextcloud Bookmarks and Nextcloud News.

Also, @szotsaki is correct. Having a proper way to "force" an update on the local cached data and/or a "clear" option is needed. I wouldn't mind having to resync / re-fetch the few megabytes of data that would need to be if everything was cleared locally.

**Comment by AndyScherzinger** [comment]
@ThomasHineXYZ agreed, makes sense in terms of having a way for cleaning and clearing, even when at the cost of re-downloading.

Cc @tobiasKaminsky @alperozturk96

**Comment by mormegil-cz** [comment]
While not a single tap, navigating to "On this device", long-pressing on an entry, selecting .../Select all and then .../Delete (and then of course local only) was quite easy, I was afraid it would be more hassle.

**Comment by btittelbach** [comment]
pitching in to underline and clarify the problem

# State

- Android 16
- Nextcloud App version 3.34.0
- I have 5 Nextcloud Accounts configured in the App
- Clearing "Internal Storage" via Android-provided App Settings would delete all accounts and creates a lot of work. For some people, this might mean visiting several admins. In my case, it means a lot of people coming to me.
- On all of these accounts:
  - The list of uploads is empty
  - All Local files have been deleted (via "On Device" Tab)
- The app (and all accounts?) are configured to use `Android/media/com.nextcloud.client` as storage location and allow other apps access.
- Calculating total space used by `Android/media/com.nextcloud.client` with a File-Manager shows 0B
  - (yes I cleaned up the `tmp` Folder)   

# Problem

Nextcloud App is still using 10GiB in "internal storage"!

# Suspicion

This storage is from uploads, where other apps "shared" files with Nextcloud and the Nextcloud App then crashed. Accumulating hidden data that the app has no record of.

# Proposal

It would be great if either/or
- the app would frequently purge it's internal orphan files
- provide the user a button to clear all internal files Without affecting account settings
- provide a way to export/import account configurations - so the android-provided "clear storage" can be used
